### DISCUSSION TITLE
Where can I find Tauri Roadmap?

### DISCUSSION THREAD

**Discussion by jerrygreen** [tauri-apps/tauri#5041] [Q&A] [answered]
I heard about Tauri a couple years ago, and I remember there was a neat Roadmap. Unfortunately I cannot find any Roadmap on tauri website nowadays. I remember there was a few interesting technologies listed that are going to be supported, and was quite hyped about that. Particularly, I’m interested in Deno support, which doesn’t seem to be released yet, but is it still planned?

Is it possible to see Roadmap anywhere to know what else is planned / what’s already done? The Roadmap was quite neat when it was listed on the website.

**Comment by FabianLars** [comment, marked as answer]
It's still on the main website: https://tauri.app/ (scroll a bit down).
We just compressed it a bit and only list bigger features now and removed target dates. It was too much of a hassle to keep it up-to-date otherwise.

About deno specifically: the plans haven't changed much, but there is still _much_ work to be done before we can even start thinking about implementing it.

**Reply by HuakunShen** [reply]
@FabianLars is there an estimated date for each roadmap item?

**Reply by FabianLars** [reply]
Nope, but the mobile bundler and the channel api are in v2 which we try to push to beta asap. i'd imagine one time commands could make it into it too. The rest is still unknown. Though alternative renderers is already partially solved with the egui plugin i guess 🤷

**Reply by HuakunShen** [reply]
Thank you for your work! 

Waiting for mobile support.

**Reply by ptphongkmf** [reply]
It’s been a year, and Tauri 2.0 has been out for a while now. Are there any updates on potential Deno support, or is it still far down on the roadmap?

**Comment by skj-skj** [comment]
[Tauri Roadmap](https://github.com/orgs/tauri-apps/projects/11/views/2) 

from here you can find in-depth info.

**Reply by FabianLars** [reply]
Disclaimer: I wouldn't put too much trust into that board yet, it was something we tried/try to see if it works for us but it may not always be accurate.
